tests/test_logging.py

```diff
@@ -0,0 +1,83 @@
+import datetime
+import logging
+import unittest
+
+from core_lib.helpers.logging import Logging
+
+
+class TestLogging(unittest.TestCase):
+    def test_string_logger(self):
+        string = "hello"
+
+        with self.assertLogs('TestLogging.logging_message', level='ERROR') as cm:
+            self.logging_message(string)
+        self.assertEqual(cm.output, ["ERROR:TestLogging.logging_message:log_for_test"])
+
+        with self.assertLogs('TestLogging.logging_without_message', level='INFO') as cm:
+            self.logging_without_message(string)
+        self.assertEqual(cm.output, ["INFO:TestLogging.logging_without_message:"])
+
+    def test_object_logger(self):
+        dat = datetime.date(2022, 1, 1)
+        obj = {'date': dat, 'tuple': ("fruit", "apple"), 'list': ["fruit", "apple"]}
+
+        with self.assertLogs('TestLogging.logging_message', level='ERROR') as cm:
+            self.logging_message(obj)
+        self.assertEqual(cm.output, ["ERROR:TestLogging.logging_message:log_for_test"])
+
+        with self.assertLogs('TestLogging.logging_without_message', level='INFO') as cm:
+            self.logging_without_message(obj)
+        self.assertEqual(
+            cm.output,
+            ["INFO:TestLogging.logging_without_message:"],

```

Comment:
how come the log ends with semicolon : ?

Reply:
`logging` returns this kind of log when there's nothing to display after the calling function when our `message` is empty then it produces this log

Reply:
so this is done by python the logger automatically?